docs/storage/upload/upload.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"google.golang.org/api/storage/v1"
```

Comment:
We recommend cloud.google.com/go/storage package for storage, could you switch to that?